etl-pipeline/services/grin/grin_client.py

```diff
@@ -44,59 +37,61 @@ def load_creds(self):
     def _url(self, fragment):
         return "https://books.google.com/libraries/NYPL/" + fragment
 
-    def get(self, url, filename, force=False):
-        cache_path = os.path.join(self.cache_dir, filename)
-        if os.path.exists(cache_path) and not force:
-            return open(cache_path, "rb").read()
+    def get(self, url, force=False):
         url = self._url(url)
```

Comment:
nit: slightly more readable
```suggestion
        url = self._url(fragment)
```

Reply:
https://github.com/NYPL/digital-research-books/pull/744/commits/de81186ed276278aa91cc23f4910f72fcf7ddcb1